backend/de.metas.business/src/main/java-legacy/org/compiere/model/CalloutOrder.java

```diff
@@ -331,13 +331,18 @@ public String bPartnerLocation(final ICalloutField calloutField)
 			orderBL.setBPLocation(order, bpartner);
 		}
 
-		if (!orderBL.setBillLocation(order))
+
+		if(order.getBill_Location_ID() <= 0)
 		{
```

Comment:
@chemsnouioua I am not ok with this. Pls debug here `de.metas.order.impl.OrderBL#setBillLocation` why this set to null and to the proper address.

Reply:
@cristinamghita 
After the change of the delivery address, the callout  `org.compiere.model.CalloutOrder#bPartnerLocation` starts.
When de.metas.order.impl.OrderBL#setBillLocation whe do not calculate nor pass the `BillToAddress` :eye: screenshot
![image](https://user-images.githubusercontent.com/438169/221281825-8163f90f-75c6-4cbd-9dcf-93b0d224c35c.png)

imho, my fix is still valid, try to get it only when its  null

Reply:
@chemsnouioua 
Yes, we do it. Here: `de.metas.document.location.adapter.IDocumentBillLocationAdapter#setFrom(de.metas.document.location.DocumentLocation)`

Reply:
replied with DM

Reply:
@chemsnouioua See my fix. Please, test it. I did not!

Reply:
works fine, thx @cristinamghita